source/guides/guides/network-requests.md

```diff
@@ -358,31 +314,30 @@ cy.get('#results')
 **Examples**
 
 ```javascript
-cy.server()
 // spy on POST requests to /users endpoint
-cy.route('POST', '/users').as('new-user')
+cy.http('POST', '/users').as('new-user')
 // trigger network calls by manipulating web app's user interface, then
 cy.wait('@new-user')
-  .should('have.property', 'status', 201)
+  .should('have.property', 'response.statusCode', 201)
 
-// we can grab the completed XHR object again to run more assertions
+// we can grab the completed request object again to run more assertions
 // using cy.get(<alias>)
-cy.get('@new-user') // yields the same XHR object
-  .its('requestBody') // alternative: its('request.body')
-  .should('deep.equal', {
+cy.get('@new-user') // yields the same request object
+  .its('request.body')
+  .should('deep.equal', JSON.stringify({
     id: '101',
     firstName: 'Joe',
     lastName: 'Black'
-  })
+  }))
```

Comment:
This example doesn't work either.

```js
it('cy.http() - route responses to matching requests', () => {
  cy.visit('https://example.cypress.io/commands/network-requests')

  cy.http('POST', '/comments').as('postComment')
  cy.get('.network-post').click()

  cy.wait('@postComment')
    .should('have.property', 'response.statusCode', 200)

  cy.get('@postComment') // yields the same request object
    .its('request.body')
    .should('deep.equal', JSON.stringify({
      id: '101',
      firstName: 'Joe',
      lastName: 'Black',
    }))
```

<img width="852" alt="Screen Shot 2020-11-20 at 2 30 47 PM" src="https://user-images.githubusercontent.com/1271364/99775016-00b82680-2b3d-11eb-9e60-46cc612f9406.png">